### PR TITLE
Add Wallet Selector - MetaMask and WalletConnect

### PR DESCRIPTION
**Pull request by danieledamiani**

### CODE REVIEW

File: packages/tempus-client_v2/src/components/wallet/UserWallet.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+type UserWallet = 'MetaMask' | 'WalletConnect';
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/wallet/UserWallet.ts:1]: This type definition file should probably go to interfaces directory.

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+            }));
+        });
+      }
+    };
+    if (!active) {
+      checkConnection();
+    }
+  }, [active, setWalletData, activate, requestNetworkChange]);
+
+  useEffect(() => {
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/wallet/Wallet.tsx:225]: What happens if user is connected using WalletConnect and MetaMask is not installed, this useEffect will set `userWalletConnected` to `false`, right?

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+              <Spacer size={10} />
+            </>
+          )}
+
+          {!connecting && selectedWallet && active && (
+            <div className="tc__connect-wallet-button" onClick={onSelectWallet}>
+              <AccountBalanceWalletIcon />
+              <Spacer size={4} />
+              {pendingTransactions.length === 0 && (
+                <Typography variant="h5">{active ? shortenedAccount : getText('connectWallet', language)}</Typography>
```
Comment (lazarmitic) [inline packages/tempus-client_v2/src/components/wallet/Wallet.tsx:291]: `active` flag will always be true here, no need to check for it, always show `shortenedAccount`.

**Review by lazarmitic — CHANGES_REQUESTED** [review]

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx (hunk 3 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:291]: right, fixed

File: packages/tempus-client_v2/src/components/wallet/UserWallet.ts (hunk 1 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/UserWallet.ts:1]: moved

File: packages/tempus-client_v2/src/components/wallet/Wallet.tsx (hunk 2 above)
Comment (danieledamiani) [reply packages/tempus-client_v2/src/components/wallet/Wallet.tsx:225]: actually this bit is not needed, removed

**Review by lazarmitic — APPROVED** [review]

**Pull request merged**